ecocodejava.properties

```diff
@@ -1,26 +1,26 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
-homepageUrl=https://github.com/green-code-initiative/ecoCode
-publicVersions=1.4.3
-archivedVersions=1.2.1,1.3.0,1.4.2
+homepageUrl=https://github.com/green-code-initiative/ecoCode-java
+publicVersions=1.5.0
+archivedVersions=1.2.1,1.3.0,1.4.2,1.4.3
 
 defaults.mavenGroupId=io.ecocode
 defaults.mavenArtifactId=ecocode-java-plugin
 
 1.2.1.description=Initial Marketplace publication of plugin for SonarQube 9.9.+ LTS
-1.2.1.sqVersions=[9.9,10.1]
+1.2.1.sqVersions=[9.9,LATEST]
```

Comment:
Okay, this is my fault. I didn't read the file wrong. I thought that 1.2.1 was the version you were archiving. Only one version can be compatible with `LATEST` at a time. Put these old archived versions back the way they were.